compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/syntax/tree/SyntaxKind.java

```diff
@@ -184,6 +185,7 @@
     SERVICE_BODY(3023),
     ANNOTATION(3024),
     METADATA(3025),
+    NIL_LITERAL(3026),
```

Comment:
Couldn't place this under section where other literals are declared as `parserTestutils.java` has the following method.
```public static boolean isTerminalNode(SyntaxKind syntaxKind) {
        return SyntaxKind.IMPORT_DECLARATION.compareTo(syntaxKind) > 0 || syntaxKind == SyntaxKind.EOF_TOKEN;
    }```